numerics/hermite3.hpp

```diff
@@ -33,6 +36,17 @@ class Hermite3 final {
   // The result is sorted.
   BoundedArray<Argument, 2> FindExtrema() const;
 
+  // |samples| must be a container; |get_argument| and |get_value| on the
+  // elements of |samples| must return |Argument| and |Value| respectively
+  // (possibly by reference or const-reference)
+  // Returns the largest error (in the given |norm|) between this polynomial and
+  // the given |samples|.
+  template<typename Samples, typename GetArgument, typename GetValue>
+  typename Normed<Difference<Value>>::NormType LInfinityError(
```

Comment:
Would the infinity symbol work in an identifier?

Reply:
No, we had the same issue with InfinitePast and InfiniteFuture.